### DISCUSSION TITLE
Support for isDoublePressing()

### DISCUSSION THREAD

**Discussion by zjwhitehead** [bxparks/AceButton#99] [Ideas]
I'd like to have a countdown on the screen as the user is holding down a button to activate a feature. 
I currently have the `setLongPressDelay` to a few seconds. 
Is there a way to expose "progress" to let the user know: 
1) that they are holding down the button but not long enough
2) how much time is left to hold it down?

Perhaps there is a simpler way of accomplishing this but it seems like a useful feature for this lib.

**Comment by bxparks** [comment]
I think you can accomplish this by listening to 2 events:

* kEventPressed
* kEventLongPressed

Start your countdown upon kEventPressed. You probably need to listen for kEventReleased to stop your countdown.
